apps/passport-client/pages/index.tsx

```diff
@@ -173,39 +173,37 @@ function RouterImpl(): JSX.Element {
           <Route path="terms" element={<NewTermsScreen />} />
         </Route>
         <Route path="/">
-          <Route path="terms" element={<TermsScreen />} />
-          <Route index element={<HomeScreen />} />
-          <Route path="login" element={<LoginScreen />} />
-
-          <Route path="components" element={<ComponentsScreen />} />
+          <Route index element={<NewHomeScreen />} />
+          <Route path="login" element={<NewLoginScreen />} />
+          <Route path="new-passport" element={<NewPassportScreen2 />} />
           <Route
-            path="login-interstitial"
-            element={<LoginInterstitialScreen />}
+            path="enter-confirmation-code"
+            element={<NewEnterConfirmationCodeScreen />}
           />
+          <Route path="create-password" element={<NewCreatePasswordScreen />} />
           <Route
             path="already-registered"
-            element={<AlreadyRegisteredScreen />}
+            element={<NewAlreadyRegisteredScreen />}
           />
-          <Route path="sync-existing" element={<SyncExistingScreen />} />
-          <Route path="privacy-notice" element={<PrivacyNoticeScreen />} />
-          <Route path="create-password" element={<CreatePasswordScreen />} />
-          <Route path="change-password" element={<ChangePasswordScreen />} />
-          <Route path="change-email" element={<ChangeEmailScreen />} />
-          <Route path="add-email" element={<AddEmailScreen />} />
-          <Route path="remove-email" element={<RemoveEmailScreen />} />
```

Comment:
if you're commenting out the imports for these functions, we should also just comment out these routes rather than deleting them altogether, otherwise delete both the routes and the imports

Reply:
https://linear.app/0xparc-pcd/issue/0XP-1476/code-comment-cleanup